db/migrate/20230921021241_add_user_id_to_tweets.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToTweets < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :tweets, :user, null: false, foreign_key: true
```

Comment:
if you wanted to recreate the column as a reference you should have deleted the existing user_id column

Got this error 

```
PG::DuplicateColumn: ERROR:  column "user_id" of relation "tweets" already exists
/home/dev0/projects/ror_training_project/week2/daniel-lopez/db/migrate/20230921021241_add_user_id_to_tweets.rb:3:in `change'

Caused by:
ActiveRecord::StatementInvalid: PG::DuplicateColumn: ERROR:  column "user_id" of relation "tweets" already exists
/home/dev0/projects/ror_training_project/week2/daniel-lopez/db/migrate/20230921021241_add_user_id_to_tweets.rb:3:in `change'

Caused by:
PG::DuplicateColumn: ERROR:  column "user_id" of relation "tweets" already exists
/home/dev0/projects/ror_training_project/week2/daniel-lopez/db/migrate/20230921021241_add_user_id_to_tweets.rb:3:in `change'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)

```